README.md

```diff
@@ -25,6 +25,20 @@ optional arguments:
                         Version (default=latest)
 ```
 
+Before the script can be executed the first time, the required dependencies have to be installed via pip (should be executed from the root of the repository):
+
+```bash
+pip install -r requirements.txt 
+```
+
+Generate the config like this (should be executed from the root of the repository):
+```bash
+python3 generate_config.py -r keptn-sandbox/datadog-service -v 0.2.0
+```
+
+The script was tested with Python 3.
```

Comment:
Let's be more precise here, and mention the actual Python version it was tested with (just not the OS/Ubuntu version).